envpool/sokoban/astar_log.cc

```diff
@@ -33,11 +33,14 @@ void RunAStar(const std::string& level_file_name,
   std::ifstream log_file_in(log_file_name);
   // check if the file is empty
   if (log_file_in.peek() == std::ifstream::traits_type::eof()) {
-    log_file_out << "Level, Actions, Steps, SearchSteps" << std::endl;
+    log_file_out << "Level,Actions,Steps,SearchSteps" << std::endl;
   } else {  // skip levels that have already been run
     std::string line;
     std::getline(log_file_in, line);  // skip header
     while (std::getline(log_file_in, line)) {
```

Comment:
It *should* be the case that line number = level number, but are we sure? Especially if sevearal concurrent workers are run. Maybe you should assert that?

Reply:
this file is only used to sequentially get the logs for the levels in a single file. So there shouldn't be any concurrent workers working on the same file.